etcdserver/api/v3rpc/util.go

```diff
@@ -49,7 +49,7 @@ func togRPCError(err error) error {
 	case etcdserver.ErrTooManyRequests:
 		return rpctypes.ErrTooManyRequests
 
-	case etcdserver.ErrNoLeader:
+	case etcdserver.ErrNoLeader, rpctypes.ErrNoLeader: // ErrNoLeader from leasehttp
```

Comment:
@heyitsanthony 

https://github.com/gyuho/etcd/blob/162069394b103a489357a76c6584c7eb28a2b70b/etcdserver/api/v3rpc/lease.go

Where does it call leasehttp.X in this file?